torch/csrc/jit/passes/quantization.cpp

```diff
@@ -365,8 +366,12 @@ void InsertObserversHelper::insertObservers(
   script::Method method = module.get_method(method_name);
   auto graph = method.graph();
   ConstantPropagation(graph);
+  // To cleanup traced graph
+  ConstantPooling(graph);
   // must do constant propagation first before replacement
   replaceConvolutionWithConv2d(graph);
+  // fuse exploded linear into aten::linear
```

Comment:
What do you mean by exploded?

Reply:
exploded means: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/passes/quantization_patterns.h#L25-L64

Reply:
I think "decomposed" is the more common term in the JIT code. e.g. `decompose_ops.cpp`

Reply:
OK